### PR TITLE
Update remote-help.md

### PR DESCRIPTION
**Pull request by msbemba**
Request to revert the previous update as the customer claims remote help is still not available on GCC.

@Smritib17  Request to review and update the proposed changes

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

Issue#https://github.com/MicrosoftDocs/memdocs/issues/3199

File: memdocs/intune/remote-actions/remote-help.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The Remote help app supports the following capabilities:
 > [!NOTE]
 > Remote help has the following limitations:  
 >
-> - Remote help is not supported on GCC High or DoD Tenants.
+> - Remote help is not supported on GCC, GCC High or DoD Tenants.
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/remote-help.md:80]:
```suggestion
> - Remote help is not supported on GCC, GCC High, or DoD Tenants.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Smritib17 Please review and signoff this PR. Thanks

**Comment by Smritib17** [comment]
@yogkumgit this item has been addressed in https://github.com/MicrosoftDocs/memdocs-pr/pull/9036
Lets close this issue

**Pull request closed without merging**